invoke/program.py

```diff
@@ -804,6 +806,27 @@ def print_task_help(self, name: str) -> None:
             print(self.leading_indent + "none")
             print("")
 
+        print("Pre-tasks:")
```

Comment:
Instead of displaying `none`, can the sections "Pre-tasks" and "Post-tasks" be made optional if the task doesn't define pre- and post- tasks respectively?

Reply:
@antoinebrl sure, on the other hand, other sections have similar behavior with 'none' in the output. Won't it be confusing to have this difference?